### DISCUSSION TITLE
Doris BE 似乎有内存泄漏问题 memory leak

### DISCUSSION THREAD

**Discussion by LCehoennardo** [apache/doris#18680] [A - General / Q&A]
如下图，最近一段时间内存在逐步上涨，重启后下降了，但依然在持续上涨
<img width="892" alt="image" src="https://user-images.githubusercontent.com/41560874/232010753-f6b6433b-4250-4993-8962-59689a244948.png">

**Comment by LCehoennardo** [comment]
doris verison 1.2.3

**Comment by yuanyuan8983** [comment]
Hello, please add my WeChat account. Let's take a look at this issue together. My WeChat account is: cyllyy810222

**Reply by chuanbowang** [reply]
请问有找到内存上涨的原因吗?我这边也有同样的问题，doris版本：doris-1.2.2-rc01
![image](https://github.com/apache/doris/assets/21078413/a5662641-dcaa-4dcf-82b7-0860d3f07c8a)

**Reply by fdf1779** [reply]
http://be_host:webserver_port/mem_tracker 可以看一下内存被什么占用了。
我之前1.2.x 也出现了这个问题
场景：需要将３－４ＴＢ的数据存到Dｏｒｉｓ中用作查询，使用的BｒｏｋｅｒLｏａｄ进行数据同步，由于数据太大，调大了max_segment_num_per_rowset 参数，内存都被用来进行小版本合并了。

**Comment by LKK1535977706** [comment]
@LCehoennardo 请问这个问题解决了吗，我也遇到了同样的问题

**Reply by maben996** [reply]
请问这里的内存占用监控包括cache部分使用的吗？

**Reply by yagagagaga** [reply]
https://doris.apache.org/zh-CN/docs/dev/admin-manual/maint-monitor/monitor-alert

**Comment by liuanxin** [comment]
被内存问题折磨的要吐血了, 2.0-beta 版本, 32G 1fe + 1be 的机器, 6 个表, 最小的表百万, 最大的表也不到 4 千万, 每隔 10 多分钟 be 就被操作系统杀一次, release notes 里面「彻底告别 OOM」的语气更像是一记响亮的耳光

**Reply by peak-xu** [reply]
我用的2.0.0正式版 也存在同样的问题,内存一直涨,不释放

**Reply by LemonLiTree** [reply]
Upgrade to 2.0.1

**Reply by wenyiliu** [reply]
有解决吗？我们最近也遇到了类似的问题，只能靠不断地重启

**Comment by dengyangguang** [comment]
请问解决了吗？也有同样的问题

**Comment by LKK1535977706** [comment]
没有解决，我定时重启集群了，这样过一段时间又继续重启



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年11月14日(周二) 中午11:54
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [apache/doris] Doris BE 似乎有内存泄漏问题 memory leak (Discussion #18680)




 
请问解决了吗？也有同样的问题

**Comment by sophiesun333** [comment]
我也遇到通用的问题，doris_be 一直增加，请问如何解决？

**Comment by Raven888888** [comment]
一样
